packages/permit/Cargo.toml

```diff
@@ -12,10 +12,12 @@ keywords = ["secret-network", "secret-contracts", "secret-toolkit"]
 
 [dependencies]
 serde = "1.0"
+ripemd160 = { version = "0.9.1", default-features = false }
+# secp256k1 = { package = "libsecp256k1", version = "0.7.0", default-features = false, features = ["static-context", "hmac-drbg"]}
+secp256k1 = "0.21.2"
```

Comment:
If `secret_toolkit::crypto::secp256k1` uses this same library, could we instead use the `secret_toolkit::crypto` abstraction, instead of using it directly?